modules/apps/push-notifications/push-notifications-web/src/main/java/com/liferay/push/notifications/web/internal/portlet/PushNotificationsPortlet.java

```diff
@@ -65,7 +65,7 @@ public void render(
 
 		renderRequest.setAttribute(
```

Comment:
Hey @GabrielHenriquedev, just remove the setAttribute call since the class is a Util now. Thx

Reply:
Also, please verify if the `PushNotificationsWebKeys.RESOURCE_BUNDLE_LOADER_PROVIDER` is in use. If not, remove the constant as well.